transfer/transfer.go

```diff
@@ -209,5 +209,8 @@ func initCoreAdaptersIfRequired() {
 	// That's why this isn't in an init() block
 	initCoreOnce.Do(func() {
 		ConfigureCustomAdapters()
+		if !config.Config.TusTransfersAllowed() {
```

Comment:
Maybe we could add a small note that this is strictly for the darkship in the 1.3 release?